src/server/routes.pl

```diff
@@ -790,17 +790,22 @@
             ->  JSON_Options = [width(0)]
             ;   JSON_Options = []),
 
+            % Compress is the logical negation of Prefixed.
```

Comment:
It really isn't. prefixed means we turn a long string 'http://foo/bar' into 'foo:Bar'. So that is actually a compression. For the cases where something lives under `@base` or `@schema`, in the proper contexts, we can completely omit the prefix, because it is the default.

Reply:
Thanks. I reverted that change. But I kept the rename of `Prefixed` to `Compress` because it was confusing to have different arg names in different files.